x-pack/plugins/features/server/feature_schema.ts

```diff
@@ -116,6 +116,21 @@ const kibanaPrivilegeSchema = schema.object({
     read: schema.arrayOf(schema.string()),
   }),
   ui: listOfCapabilitiesSchema,
+  replacedBy: schema.maybe(
```

Comment:
should we add `schema.conditional` here? 
Since we allow to define it only when deprecated field is present

Reply:
@elena-shostak, the last time I worked with `schema.conditional`, it only allowed using `siblingRef` for direct siblings. Is there a way to use conditional with something that isn’t a direct sibling (e.g., top-level `feature.deprecated` vs nested `feature.all.replacedBy` or `feature.subFeatures.[].replacedBy`)?

Reply:
Checked that out, unfortunately, it indeed works only with `siblingRef`

Reply:
That's unfortunate, but thanks for checking, @elena-shostak!